diff -uNr GwifiMeter.orig/callback.c GwifiMeter/callback.c
--- GwifiMeter.orig/callback.c	2006-05-02 09:57:45.000000000 +0200
+++ GwifiMeter/callback.c	2006-05-08 13:51:23.000000000 +0200
@@ -176,7 +176,7 @@
 char *user;
 user = readfunc("whoami",NULL,NULL,1);
 pdata->user = g_realloc(pdata->user,strlen(user)+19);
-strcpy(pdata->user,"/home/");
+strcpy(pdata->user,"/home/users/");
 strcat(pdata->user,user);
 strcat(pdata->user,"/.gwifimeter");
 //printf("%s\n",pdata->user);
